### PR TITLE
Sites api documentation

### PR DESCRIPTION
**Pull request by tinashechiraya**
Description:

Documentation on how to use the sites api

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 50 sec

File: docs/src/developer/sites.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+#### Endpoint: `https://{current_domain}/monitor/sites/`
+
+Fields required for site creation:
+- `the_geom`: Geometric point field (e.g., SRID=4326;POINT (24.84165007535725 -30.47829136066817))
+- `site_name`: Name of the site (max length: 15, mandatory)
+- `river_name`: Name of the river (max length: 15, mandatory)
+- `description`: Description of the site (max length: 255)
+- `river_cat`: River category (choices: 'rocky', 'sandy')
+- `user`: User reference
+- `time_stamp`: Optional field ,if not provided current datetime is added to the site
```
Comment (Jeremy-Prior) [inline docs/src/developer/sites.md]:
```suggestion
- `time_stamp`: Optional field, if not provided current datetime is added to the site
```

File: docs/src/developer/sites.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    "user": 1
+}
+```
+
+
+### 4. Update a Site
+
+#### Endpoint: `/sites/<site_id>/` (e.g., `https://{current_domain}/monitor/sites/1/`)
+
+any field can be updated on the site
```
Comment (Jeremy-Prior) [inline docs/src/developer/sites.md]:
```suggestion
Any field can be updated on the site
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]
Minor changes

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 9 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 18 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 5 sec

**Review by Jeremy-Prior — APPROVED** [review]
LGTM

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 46 sec

**Pull request merged**